addons/l10n_br/data/l10n_latam.document.type.csv

```diff
@@ -0,0 +1,32 @@
+id,sequence,code,country_id/id,name,internal_type,doc_code_prefix,active
+dt_01,10,01,base.br,Invoice 1 / 1A,"",NF,False
+dt_1B,20,1B,base.br,Single invoice,"",NFA,False
+dt_02,30,02,base.br,In-Consumer Sales Invoice,"",NFC,False
+dt_2D,40,2D,base.br,Tax Coupon,invoice,CF,False
+dt_2E,50,2E,base.br,Tax Coupon-Ticket,invoice,TIQP,False
+dt_04,60,04,base.br,Invoice from Producer,invoice,NFP,False
+dt_06,70,06,base.br,Nota Fiscal / Electricity Bill,invoice,NFCE,False
+dt_07,80,07,base.br,Invoice for Transport Service,invoice,NFST,False
+dt_08,90,08,base.br,Ground Bill of Lading,invoice,CTRC,False
+dt_8B,100,8B,base.br,Loose Ground Bill of Lading,invoice,CTCA,False
+dt_09,110,09,base.br,Maritime Bill of Lading,invoice,CTAC,False
+dt_10,120,10,base.br,Aircraft Knowledge,invoice,CA,False
+dt_11,130,11,base.br,Railway Bill of Lading,invoice,CTFC,False
+dt_13,140,13,base.br,Road Ticket,invoice,BPR,False
+dt_14,150,14,base.br,Waterway Ticket,invoice,BPA,False
+dt_15,160,15,base.br,e-Baggage Ticket,invoice,BPNB,False
+dt_16,170,16,base.br,Railway Ticket,invoice,BPF,False
+dt_18,180,18,base.br,Daily Movement Summary,invoice,RMD,False
+dt_21,190,21,base.br,Invoice for de-Communication Service,invoice,NFSC,False
+dt_22,200,22,base.br,Invoice for Telecommunication Service,invoice,NFST,False
+dt_26,210,26,base.br,Bill of Lading Multimodal Transport,invoice,CTMC,False
+dt_27,220,27,base.br,Invoice for Rail Transport De-Cargo,invoice,NFTFC,False
+dt_28,230,28,base.br,Invoice / Gas Supply Channel Account,invoice,NFCFGC,False
+dt_29,240,29,base.br,Invoice / Water Supply Account,invoice,NFCFAC,False
+dt_55,250,55,base.br,Electronic Invoice (NF-e),"",NFe,True
+dt_57,260,57,base.br,Electronic Bill of Lading (CT-e),invoice,CTe,False
+dt_58,270,58,base.br,Electronic Manifesto of Tax Documents (MDF-e),invoice,MDFe,False
+dt_59,280,59,base.br,Electronic Tax Coupon (CF-e-SAT),invoice,CFeS,False
+dt_60,290,60,base.br,Electronic Tax Coupon (CF-e-ECF),invoice,CFeE,False
+dt_65,300,65,base.br,Electronic Invoice to the Final Consumer (NFC-e),"",NFCe,True

```

Comment:
@jco-odoo yeah, you are right... maybe it will be better to have the electronic ones at the beginning, we just follow the order given by Avalara on the table. But it makes sense that if you activate the non-electronic, they will come after the electronic ones.

As to registering a vendor bill, probably the most common cases will be the electronic ones.

@jorenvo I think we can allocate the electronic ones first and on top of the rest, and for the rest keep with the current order

Reply:
Ok